lib/ccf/structural_preprocessing/one_subject_run_status_checker.py

```diff
@@ -3,6 +3,8 @@
 # import of built-in modules
```

Comment:
Are all these changes just part of the submitting jobs from "local" machine feature? If so move these changes to a different branch for a separate pull/review/merge.